client/src/components/layout/TripShow.js

```diff
@@ -27,6 +32,37 @@ const TripShow = (props) => {
     showTrip()
   }, [])
 
+  const postComment = async (comment) => {
+    try {
+      const response = await fetch(`/api/v1/trips/${tripId}/comments`, {
+        method: "POST",
+        headers: new Headers({
+          "Content-Type": "application/json",
+        }),
+        body: JSON.stringify(comment),
+      })
+      if (!response.ok) {
+        if (response.status == 422) {
+          const body = await response.json()
+          const newErrors = translateServerErrors(body.errors)
+          return setErrors(newErrors)
+        }
+        const errorMessage = `${response.status} (${response.statusText})`
+        const error = new Error(errorMessage)
+        throw error
+      } else {
+        const responseBody = await response.json()
+        if (responseBody.newComment) {
+          showTrip()
```

Comment:
haha yeah this is no-beuno, even though it should work. You and I went over this! 
- that `reponseBody.newComment` needs to find its way into your `trip.comments` state...thats it! We don't need to make a new request to do that we just need 
`setTrip({...trip, comments: trip.comments.concat(responseBody.newComment`})